### PR TITLE
Create Day Loader

### PR DESCRIPTION
**Pull request by aikaniel**
This PR creates the Day Loader.

### CODE REVIEW

File: app/Loaders/DayLoader.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                ->select()
+                ->where('id', '=', $id)
+                ->first();
+        if (!$dayData) {
+            throw new RecordsNotFoundException('Day not found with id: ' . $id);
+        }
+
+        return get_object_vars($dayData);
+    }
+}
```
Comment (jtowe1) [inline app/Loaders/DayLoader.php:23]:
```suggestion
}

```

File: app/Loaders/DayLoader.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+use Illuminate\Database\RecordsNotFoundException;
+use Illuminate\Support\Facades\DB;
+
+class DayLoader
+{
+    public function loadById(int $id): array
+    {
+        $dayData =
+            DB::table('sprint')
```
Comment (jtowe1) [inline app/Loaders/DayLoader.php:13]:
```suggestion
            DB::table('day')
```

Comment (aikaniel) [reply app/Loaders/DayLoader.php:13]: WHOOPS

**Comment by jtowe1** [comment]
Let's also add a `loadByDateCode` method to the Loader, since I think we'll be looking them up by dateCode mostly.

**Review by jtowe1 — APPROVED** [review]

**Pull request merged**